ChangeLog

```diff
@@ -26,6 +26,7 @@
  * Deprecated OptimizationProblem(levelFunction, levelValue)
  * Deprecated (Linear|Quadratic)(LeastSquares|Taylor)::getResponseSurface
  * FittingTest::BestModelBIC returns the Distribution and the BIC value
+ * Add statistic attribut and accessor to the TestResult class
```

Comment:
statistic*s*! It must be changed everywhere, sorry...

Reply:
indeed...

Reply:
statistics_ ? I dont see it used at the plural anywhere, https://en.wikipedia.org/wiki/Test_statistic

Reply:
ok my bad, go for statistic_